modules/nw-dns-forward.adoc

```diff
@@ -42,10 +51,24 @@ spec:
       upstreams:
         - 3.3.3.3
         - 4.4.4.4:5454
+  upstreamResolvers: <4>
+    policy: Random <5>
+    upstreams: <6>
+    - type: SystemResolvConf <7>
+    - type: Network
+      address: 1.2.3.4 <8>
+      port: 53 <9>
 ----
 <1> `name` must comply with the `rfc6335` service name syntax.
 <2> `zones` must conform to the definition of a `subdomain` in `rfc1123`. The cluster domain, `cluster.local`, is an invalid `subdomain` for `zones`.
 <3> A maximum of 15 `upstreams` is allowed per `forwardPlugin`.
+<4> `upstreamResolvers` is optional. You can use it to override the default policy and forward DNS resolution to the specified DNS resolvers (upstream resolvers) for the default domain. If you do not provide any upstream resolvers, the DNS name queries go to the servers in `/etc/resolv.conf`.
+<5> `policy` determines the order in which upstream servers are selected for querying. You can specify one of these values: `Random`, `RoundRobin`, and `Sequential`. The default value is `Sequential`.
+<6> `upstreams` is optional. You can use it to provide upstream resolvers.
+<7> You can specify two types of `upstreams` - `SystemResolvConf` and `Network`. `SystemResolvConf` is for configuring the upstream to use `/etc/resolv.conf` and `Network` is for defining a `Networkresolver`. You can specify one or both.
+<8> If the specified type is `Network`, you must provide an IP address. `address` must be a valid IPv4 or IPv6 address.
+<9> If the specified type is `Network`, you can optionally provide a port. `port` must be between 1 and 65535.
```

Comment:
I don't think it's necessary in some of these to repeat the field name.

```suggestion
<4> Optional. You can use this setting to override the default policy and forward DNS resolution to the specified upstream DNS resolvers for the default domain. If you do not provide any upstream resolvers, the DNS name queries go to the servers in `/etc/resolv.conf`.
<5> Determines the order in which upstream servers are selected for querying. You can specify one of these values: `Random`, `RoundRobin`, and `Sequential`. The default value is `Sequential`.
<6> Optional. You can use it to provide upstream resolvers.
<7> There are two types of `upstreams`: `SystemResolvConf` configures the upstream to use `/etc/resolv.conf`, and `Network` defines a `Networkresolver`. You can specify one or both.
<8> If the specified type is `Network`, you must provide an IP address. `address` must be a valid IPv4 or IPv6 address.
<9> If the specified type is `Network`, you can optionally provide a port. `port` must be between 1 and 65535.
```

Reply:
Comment addressed.